tasks/main.yml

```diff
@@ -64,20 +65,6 @@
     group: confluence
   notify: Restart confluence service
 
-- name: Grant write privileges
-  file:
-    path: /opt/atlassian-confluence-{{ confluence_version }}/{{ item }}
-    owner: root
-    group: confluence
-    mode: 0775
-    recurse: yes
-  with_items:
-    - conf
-    - temp
-    - webapps
-    - work
-  notify: Restart confluence service
-
```

Comment:
Removing this task results in ownership `root:root` and `0755` permissions because of the deploy task
```yaml
- name: Unpack Confluence archive
  unarchive:
    src: "{{ global_cache_dir }}/confluence-{{ confluence_version }}.tar.gz"
    dest: /opt/atlassian-confluence-tmp
    owner: root
    group: root
    mode: 0755
    ```
I think we need address this there as well, otherwise confluence can't write to these directories.